src/vdaf/prio3.rs

```diff
@@ -1176,35 +1183,39 @@ mod tests {
     }
 
     #[test]
-    fn test_prio3_countvec() {
-        let prio3 = Prio3::new_aes128_count_vec(2, 20).unwrap();
+    fn test_prio3_sum_vec() {
+        let prio3 = Prio3::new_aes128_sum_vec(2, 2, 20).unwrap();
         assert_eq!(
             run_vdaf(
                 &prio3,
                 &(),
-                [vec![
-                    0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 1, 1, 1, 0, 1,
-                ]]
+                [
+                    vec![0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 1, 1, 1, 0, 1],
+                    vec![0, 2, 0, 0, 1, 0, 0, 0, 1, 1, 1, 3, 0, 3, 0, 0, 0, 1, 0, 0],
+                    vec![1, 1, 1, 0, 1, 1, 0, 1, 1, 1, 1, 0, 1, 1, 1, 1, 0, 1, 0, 1],
+                ]
             )
             .unwrap(),
-            vec![0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 1, 1, 1, 0, 1,]
+            vec![1, 3, 1, 0, 3, 1, 0, 1, 2, 2, 3, 3, 1, 5, 1, 2, 1, 3, 0, 2],
```

Comment:
@simon-friedberger here is the expanded test (Similarly for the multithreaded below.)